test/dataset.py

```diff
@@ -0,0 +1,153 @@
+import uuid
+
+NUM_CHILDREN_PER_PING = 3
+SCALAR_VALUE = 42
+
```

Comment:
It would be nice to have some instructions in the README on how to add a new dimension to the job.